.github/workflows/build-cfs.yml

```diff
@@ -1,4 +1,4 @@
-name: Build, Test, and Run [OMIT_DEPRECATED=true]
+name: Build, Test, and Run
```

Comment:
We need to be careful about name collision with the workflow that has OMIT_DEPRECATED=false

Reply:
CodeQL analysis will run if workflow name does not have brackets. Can it be named Build, Test, and Run OMIT_DEPRECATED=true?

Reply:
Escape characters \\[OMIT_DEPRECATED=true\\] works.